apps/omg_watcher/lib/block_getter/core.ex

```diff
@@ -544,6 +570,20 @@ defmodule OMG.Watcher.BlockGetter.Core do
     {:ok, %PotentialWithholdingReport{blknum: requested_number, hash: requested_hash, time: time}}
   end
 
+  defp to_block_application(block, recovered_txs, block_timestamp) do
```

Comment:
Can we make it `BlockApplication` module's function?, e.g. `BlockApplication.from(...)` or `BA.new(...)`